ipatests/test_xmlrpc/test_host_plugin.py

```diff
@@ -337,6 +341,84 @@ def test_add_host_with_null_password(self, host):
         host.track_create()
         host.check_create(result)
 
+    @staticmethod
+    def modify_host_config(host_tracker, value):
+        command = host_tracker.make_command(
+            'config_mod',
+            **dict(
+                setattr=u'maxhostnamelen={}'.format(value)))
+        command()
+
+    def test_lower_hostname_limit_below_maxhostnamelimit(self):
```

Comment:
A better name for the test would be `test_config_maxhostname_invalid` with comment """Change config maxhostname to an invalid value (lower than MAXHOSTNAMELEN)"""